pkg/kubelet/cm/cpumanager/state/state_checkpoint.go

```diff
@@ -159,37 +158,52 @@ func (sc *stateCheckpoint) SetCPUSet(containerID string, cset cpuset.CPUSet) {
 	sc.mux.Lock()
 	defer sc.mux.Unlock()
 	sc.cache.SetCPUSet(containerID, cset)
-	sc.storeState()
+	err := sc.storeState()
+	if err != nil {
+		panic("store state error: " + err.Error())
```

Comment:
I took a look elsewhere in the container manager. In the device manager, the current behavior is to log a warning when this occurs. Would you mind making this code consistent with the device manager as part of this PR? This function signature would not need to change. Same for the other four panics in this file, and also state_file.go.

Reply:
Ok, log warning is better. I've updated these codes.